.github/workflows/run-fvt.yml

```diff
@@ -87,4 +93,4 @@ jobs:
           export PATH=/root/go/bin/:$PATH
           export NAMESPACE=modelmesh-serving
           export NAMESPACESCOPEMODE=false
-          make fvt
+          make fvt
```

Comment:
missing newline at end of file

Reply:
do we need a blank line at the end?

Reply:
@ckadner could you please answer my question?

Reply:
https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file/18789#18789

Reply:
thanks for the information. I added the blank line.

Reply:
I do see an extra line now but Github still complains about the missing new line at the end of file:

![image](https://user-images.githubusercontent.com/12246093/232176181-530241df-3db6-4e29-aa47-5a0bca3211f9.png)


![image](https://user-images.githubusercontent.com/12246093/232176163-7b0089dc-9b52-48c3-ab77-89b06ccf75bc.png)

Are you working on Windows by any chance?

https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

Reply:
@ckadner interesting. could you please check this again? I set vs code setting and now I can not see the red circle anymore.

Reply:
looks good now -- Thanks